docs/reference-terminology.md

```diff
@@ -78,18 +67,6 @@ A group of namespaces that have anti-affinity to each other.
 A lightweight Pulsar broker in which all components run in a single Java Virtual Machine (JVM) process. Standalone
 clusters can be run on a single machine and are useful for development purposes.
 
-### Cluster
```

Comment:
Any background why we remove these terms?

Reply:
As we have included the definitions of certain terms in the Concepts and Architecture chapter, redundant terminology definitions would add to the maintenance complexity. Users can refer to this chapter when seeking the definitions of specific terms.

Perhaps we can place a link at the repeated term's location in "reference-terminology.md" as a substitute for those redundant definitions?

e.g.
## Producer
See [here](https://pulsar.apache.org/docs/next/concepts-clients/#producer).

On the other hand, for instance, when a user searches for "what is Pulsar brokers," we expect the result is https://pulsar.apache.org/docs/next/concepts-architecture-overview instead of https://pulsar.apache.org/docs/next/reference-terminology/. Repetitive definitions would disperse the weight of the expected page.

Reply:
I don't think it's necessary to keep those items empty and add links in the reference-terminology.md because we want to navigate users to "concept" topics only on purpose.

Reply:
Thanks for your explanation!